lib/workload/stateless/stacks/pieriandx-pipeline-manager/Readme.md

```diff
@@ -0,0 +1,247 @@
+# PierianDx Pipeline Manager
+
+Giant, serverless pipeline attached to a serverless database
```

Comment:
This does not create WRSC events, right? 
Would it make sense to add those, so the WorkflowManager can track them?

(I might have missed it)

Reply:
The 'nails' glue creates workflow run state change 'ready' events.   

The monitor-pieriandx-runs step functions will generate a workflow run state change event once pieriandx has completed